src/crate/theme/rtd/conf/__init__.py

```diff
@@ -115,7 +115,7 @@
 ogp_site_url = "https://crate.io/"
 ogp_description_length = 300
 ogp_site_name = "CrateDB documentation"
-ogp_image = "https://user-images.githubusercontent.com/453543/119536319-3c9ca480-bd89-11eb-908d-3da78e55f17b.png"
+ogp_image = "https://crate.io/hubfs/static/og_crateio.png"
```

Comment:
Shall the dimensions also be reduced?

Reply:
@amotl the 1200x630 is/was the recommended size for opengraph images (https://developers.facebook.com/docs/sharing/webmasters/images/ for reference). However, after a quick search apparently size doesn't matter as much anymore, but I reckon we should still go with those dimensions. Any other insights @michaelkremmel?

Reply:
The aspect ratio is more important than the size itself since the different social networks have different requirements and recommendations. But with the [image](https://crate.io/hubfs/static/og_crateio.png) all the recommendations are met and we are good to go here.

Those are the usual recommendations:
linkedin:
max. datasize: 5mb
max. imagesize: 7680x4320 / minimum: 640x360
aspect ratio: 1.91:1

facebook:
max. datasize: 30mb
minimum size: 1080x1080
aspect ratio: 1.91:1 - 1:1 (tolerance up to 3%)

twitter:
max. datasize: 5mb
recommended size:800x418 for aspect ratio 1,91:1